cmd/temporal-lens/main.go

```diff
@@ -68,7 +70,15 @@ var commands = map[string]cmd.Cmd{
 
 			// create lens v2 service
 			l.Info("instantiating Lens V2")
-			srv, err := lens.NewV2(lens.V2Options{}, manager, tf, l)
+			srv, err := lens.NewV2(lens.V2Options{
+				Engine: engine.Opts{
+					StorePath: *dsPath,
+					Queue: queue.Options{
+						Rate:      500 * time.Millisecond,
+						BatchSize: 1,
```

Comment:
we should move these into the Lens config file to make it easier to scale the options if needed based on how popular Lens ends up being

bare minimum change we should do right now is to make `Rate` be 5 seconds or something - 500ms is way too often, and only useful in tests where faster flushing is important

Reply:
good call, i'll open up a PR for the config changes